test/sequential/test-inspector-has-inspector-false.js

```diff
@@ -0,0 +1,20 @@
+// Flags: --expose-internals
+'use strict';
+
+const common = require('../common');
+
+// This is to ensure that the sendInspectorCommand function calls the error
+// function if its called with the v8_enable_inspector is disabled
+
+const v8EnableInspectorBk = process.config.variables.v8_enable_inspector;
+process.config.variables.v8_enable_inspector = 0;
+const inspector = require('internal/util/inspector');
+
+function restoreProcessVar() {
+  process.config.variables.v8_enable_inspector = v8EnableInspectorBk;
+}
+
+inspector.sendInspectorCommand(
+  common.mustNotCall('Inspector callback should not be called'),
+  common.mustCall(restoreProcessVar, 1),
```

Comment:
Hi, @madeinjam! Welcome and thanks for the pull request! I believe `restoreProcessVar()` can be removed from the file. The process exits, so there's no need to reset the value.

Reply:
Hi @Trott ! I made the change, thank you!